velox/connectors/hive/HiveConfig.cpp

```diff
@@ -254,4 +254,8 @@ uint8_t HiveConfig::parquetWriteTimestampUnit(const Config* session) const {
   return unit;
 }
 
+std::string HiveConfig::sessionTimezone(const Config* session) const {
+  return session->get<std::string>(kSessionTimezone, "");
```

Comment:
@Yuhta isn't it odd to add a config for a session values? Should they not be plumbed via `ConnectorQueryCtx->sessionProperties`

Reply:
@majetideepak In the `HiveConnectorUtil.configureReaderOptions` method, I obtain the timezone from `connectorQueryCtx_->sessionProperties()`.

https://github.com/facebookincubator/velox/pull/9781/files#diff-bb5108c73c856254061967ee227621c6176da13b393d419727d3a9bfdfca7764R540